proxy.live.php

```diff
@@ -18,9 +19,12 @@
 $cpanelIntegration = new CF\Integration\DefaultIntegration($config, $cpanelAPI, $dataStore, $logger);
 $partialZoneSet = new \CF\Cpanel\Zone\Partial($cpanelAPI, $dataStore, $logger);
 $requestRouter = new \CF\Router\RequestRouter($cpanelIntegration);
-$requestRouter->addRouter('\CF\API\Client', \CF\Cpanel\ClientV4APIRoutes::$routes);
-$requestRouter->addRouter('\CF\API\Plugin', \CF\Cpanel\PluginRoutes::getRoutes(\CF\API\PluginRoutes::$routes));
-$requestRouter->addRouter('\CF\API\Host', \CF\Cpanel\HostRoutes::$routes);
+$clientAPI = new \CF\API\Client($cpanelIntegration);
+$requestRouter->addRouter($clientAPI, \CF\Cpanel\ClientV4APIRoutes::$routes);
+$pluginAPI = new \CF\API\Plugin($cpanelIntegration);
+$requestRouter->addRouter($pluginAPI, \CF\Cpanel\PluginRoutes::getRoutes(\CF\API\PluginRoutes::$routes));
+$hostAPI = new \CF\API\Host($cpanelIntegration);
+$requestRouter->addRouter($hostAPI, \CF\Cpanel\HostRoutes::$routes);
```

Comment:
Magento creates all the objects dynamically, I have to use a different HTTP client, and I have to pass it in through the constructor.  The old implementation hardcoded the constructor args so it complained when Magneto tried to pass a second argument.